vllm/lora/models.py

```diff
@@ -377,8 +385,12 @@ def activate_adapter(
             module_lora = lora_model.get_lora(module_name)
             if module_lora:
                 module_lora.optimize()
+                # Bias is not explicitly enabled with the flag enable_lora_bias.
+                if not self.lora_config.bias_enabled:
+                    module_lora.bias = None
```

Comment:
So if there is an adapter with bias but it's not been enabled in the global config then we just ignore it?

Would it not be better to raise an exception in this case and fail loading of the adapter? At minimum I guess a warning should be logged.

Best of all would be to not have to configure it explicitly per my other comment, but perhaps that isn't feasible.

Reply:
Actually per my other comment [update](https://github.com/vllm-project/vllm/pull/5733#discussion_r1797489003), I think a hard failure here might be best.

Reply:
Now, I have let it fail when the adapter has bias, but is not enabled.